.github/workflows/clang-format-pull-request.yml

```diff
@@ -4,22 +4,21 @@ on:
   pull_request:
     branches:
       - release_oneDPL
-      - master
+      - main
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 1
+        fetch-depth: 0
 
     - name: Get clang-format
       run: sudo apt-get install -yqq clang-format-6.0
 
     - name: Applying clang-format for changed files
       run: |
-        URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
-        FILES=$(curl -s --request GET --get $URL |  jq -r '.[] | .filename' | grep ^include | grep -v nanorange\.hpp\$ || true)
+        FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} | grep ^include | grep -v nanorange\.hpp\$ || true)
```

Comment:
Will it work for forks?

Reply:
`github.event.pull_request.base.sha` value (base commit checksum) is set for each pipeline started for a pull request. So I think it should work for forks too